src/Http/Middleware/LockerTimer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace lockscreen\FilamentLockscreen\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+class LockerTimer
+{
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    public function handle($request, Closure $next)
+    {
+        if ($request->method() === 'GET' && $request->session()->get('locker_last_activity') && (time() - $request->session()->get('locker_last_activity')) > config('filament-lockscreen.activity_timeout', 0)) {
```

Comment:
can you change the default value to 30minutes if the `activity_timeout` doesn't exist.